<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loading2</class>
 <widget class="QMainWindow" name="Loading2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="shadow">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(45, 48, 55);
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>661</width>
         <height>181</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Casta</family>
         <pointsize>45</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #E7B18E;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Query&lt;span style=&quot; font-weight:600;&quot;&gt;-&lt;/span&gt;Document Similarity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_desc">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>190</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(110, 117, 134);</string>
       </property>
       <property name="text">
        <string>Menghitung kemiripan beberapa dokumen dengan query</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>601</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar{
	background-color: rgb(110, 117, 134);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius:10px;
	text-align: center;
}

QProgressBar::chunk{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.46, x2:1, y2:0.46, stop:0 rgba(231, 177, 142, 255), stop:1 rgba(138, 113, 99, 255));
	border-radius: 10px;
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>310</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(110, 117, 134);</string>
       </property>
       <property name="text">
        <string>loading...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_credits">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gramatika-Medium</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(110, 117, 134);</string>
       </property>
       <property name="text">
        <string>Final project by : Naufal Mufid F</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
